### DISCUSSION TITLE
Welcome to Stock-Price-Prediction-Using-LSTM Discussions!

### DISCUSSION THREAD

**Discussion by 034adarsh** [034adarsh/Stock-Price-Prediction-Using-LSTM#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jaswant05** [comment]
requirements.txt file is missing

**Reply by 034adarsh** [reply]
Hi Jaswant, I have updated the readme file. No requirement.txt file is needed. Just download the notebook and choose the company stock you would like to work on. The code will fetch data directly using the Yahoo Finance library. Just write the correct stock name in the code.
